x-pack/plugins/apm/scripts/test/api.js

```diff
@@ -33,9 +33,14 @@ const { argv } = yargs(process.argv.slice(2))
     description:
       'Run all tests (an instance of Elasticsearch and kibana are needs to be available)',
   })
+  .option('updateSnapshots', {
+    default: false,
+    type: 'boolean',
+    description: 'Update all Snapshots ',
+  })
   .help();
```

Comment:
If arguments are forwarded we wont need this, right?

Reply:
yes!